docs/guides/dataloaders.md

```diff
@@ -295,7 +295,7 @@ class MyGraphQL(GraphQL):
 @strawberry.type
 class Query:
     @strawberry.field
-    async def get_user(self, info: strawberry.Info, id: strawberry.ID) -> User:
+    async def get_user(self, info: Info, id: strawberry.ID) -> User:
```

Comment:
Info is available on strawberry in the latest version 😊

Reply:
ah, I see. Could be easier if we had strawberry version selector in the documentation.